package.json

```diff
@@ -52,7 +52,8 @@
     "cpm"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.7"
+    "ember-cli-babel": "^5.1.7",
+    "ember-macro-helpers": "kellyselden/ember-macro-helpers#ember-cpm"
```

Comment:
I had to make an API addition to support this refactor. If you approve, I will release a new version before merge.